### PR TITLE
Update AutoHeal.java and Commands.java

### PR DESCRIPTION
**Pull request by qloha**
## Summary by CodeRabbit

- **New Features**
	- Enhanced the AutoHeal module with new toggle settings for healing items: goldenHead, soup, and goldenHeadName.
	- Streamlined the healing item selection process for improved user experience.
	- Updated command handling for the bind command to allow case-insensitive module name input and standardized key code processing.

- **Bug Fixes**
	- Simplified item dropping logic to ensure smoother gameplay when using soup.

### CODE REVIEW

**Comment by qloha** [comment]
i forgot to merge into dev again 😂

File: src/main/java/keystrokesmod/module/impl/player/AutoHeal.java [hunk 1]
```diff
@@ -16,20 +14,25 @@
...
+
     public AutoHeal() {
         super("AutoHeal", category.player);
-        this.registerSetting(new DescriptionSetting("help you win by auto use healing item."));
-        this.registerSetting(item = new ModeSetting("Item", new String[]{"Golden head", "Soup"}, 0));
-        this.registerSetting(autoThrow = new ButtonSetting("Auto throw", false, new ModeOnly(item, 1)));
+        this.registerSetting(new DescriptionSetting("Automatically uses healing items."));
+        this.registerSetting(goldenHead = new ButtonSetting("Golden head", true));
+        this.registerSetting(goldenHeadName = new ButtonSetting("Check golden Head name", true, goldenHead::isToggled));
+        this.registerSetting(soup = new ButtonSetting("Soup", false));
```
[the remark this answers is not in the document]
Comment (qloha) [reply src/main/java/keystrokesmod/module/impl/player/AutoHeal.java:34]: no i did that for a reason

File: src/main/java/keystrokesmod/module/impl/player/AutoHeal.java [hunk 2]
```diff
@@ -39,18 +42,32 @@
...
+                                    break;
+                                }
+
+                            } else {
+                                toSlot = slot;
+                                break;
+                            }
+                        }
+                    }
+                }
```
[the remark this answers is not in the document]
Comment (qloha) [reply src/main/java/keystrokesmod/module/impl/player/AutoHeal.java:66]: bro what

File: src/main/java/keystrokesmod/module/impl/player/AutoHeal.java [hunk 3]
```diff
@@ -82,7 +99,11 @@
                 lastSwitchTo = -1;
                 lastDoneUse = -1;
                 lastHeal = System.currentTimeMillis();
+
+                if (mc.thePlayer.getHealth() <= minHealth.getInput()) {
+                    lastSwitchTo = -1;
+                }
```
[the remark this answers is not in the document]
Comment (qloha) [reply src/main/java/keystrokesmod/module/impl/player/AutoHeal.java:105]: i didnt have any issues when testing

**Review by xia-mc — CHANGES_REQUESTED** [review]
Please fork from ddev branch

**Comment by qloha** [comment]
new

**Pull request closed without merging**